registry/tracecat_registry/base/core/email.py

```diff
@@ -35,13 +36,30 @@ def _build_email_message(
     recipients: list[str],
     subject: str,
     body: str,
+    content_type: str | None = None,
     bcc: str | list[str] | None = None,
     cc: str | list[str] | None = None,
     reply_to: str | list[str] | None = None,
     headers: dict[str, str] | None = None,
 ) -> EmailMessage:
     msg = EmailMessage()
-    msg.set_content(body)
+
+    # Handle content based on content_type
+    if content_type == "text/html":
+        # Follow MIME standards for HTML emails
+        # https://www.w3.org/Protocols/rfc1341/7_2_Multipart.html
+        msg.add_alternative(
+            "This email requires an HTML viewer to display properly.", subtype="plain"
+        )
+        sanitized_body = nh3.clean(body)
+        msg.add_alternative(sanitized_body, subtype="html")
+    elif content_type == "text/plain":
+        msg.set_content(body)
+    else:
+        raise ValueError(
+            f"Unsupported content type: {content_type}. Expected 'text/plain' or 'text/html'."
+        )
```

Comment:
if `content_type == None` it will fallthrough here - i know there's defaults defined in the UDF but the `_build_email_message` method will always raise since its defined optional. Maybe handle the optional case in the body?

Reply:
Made it required